### ISSUE TITLE
typescriptOptions System.config required?

### ISSUE THREAD

**Issue by peteruithoven** [frankwallis/plugin-typescript#27]
When I don't add a `typescriptOptions` to `System.config` I get the following error. 
```
System.typescriptOptions is undefined
```
Adding a empty object (see below) is enough, but couldn't this be made optionally? 

``` javascript 
System.config({
  'typescriptOptions': {}
});
```

**Comment by frankwallis** [comment]
Thanks for raising this, it's a bug and I'll put out a fix shortly.

**Comment by frankwallis** [comment]
fixed in 2.0.2
